source/backend/api/Services/ProjectService.cs

```diff
@@ -28,29 +31,43 @@ public class ProjectService : BaseService, IProjectService
         /// <param name="projectRepository"></param>
         /// <param name="productRepository"></param>
         /// <param name="acquisitionFileRepository"></param>
+        /// <param name="userRepository"></param>
+        /// <param name="lookupRepository"></param>
         public ProjectService(
             ClaimsPrincipal user,
             ILogger<ProjectService> logger,
             IProjectRepository projectRepository,
             IProductRepository productRepository,
-            IAcquisitionFileRepository acquisitionFileRepository)
+            IAcquisitionFileRepository acquisitionFileRepository,
+            IUserRepository userRepository,
+            ILookupRepository lookupRepository)
             : base(user, logger)
         {
             _logger = logger;
             _projectRepository = projectRepository;
             _productRepository = productRepository;
             _acquisitionFileRepository = acquisitionFileRepository;
+            _userRepository = userRepository;
+            _lookupRepository = lookupRepository;
             _user = user;
         }
 
         public IList<PimsProject> SearchProjects(string filter, int maxResult)
         {
-            _logger.LogInformation("Getting all projects");
+            _logger.LogInformation("Searching for projects that match {filter}", filter);
             _user.ThrowIfNotAuthorized(Permissions.ProjectView);
 
-            var projects = _projectRepository.SearchProjects(filter, maxResult);
+            // Limit search results to user's assigned region(s), but always include "Cannot determine" region
+            var pimsUser = _userRepository.GetUserInfoByKeycloakUserId(_user.GetUserKey());
+            var userRegions = pimsUser.PimsRegionUsers.Select(r => r.RegionCode).ToHashSet();
 
-            return projects;
+            var noRegion = _lookupRepository.GetAllRegions().FirstOrDefault(r => r.RegionName == "Cannot determine")?.Code;
+            if (noRegion.HasValue)
+            {
+                userRegions.Add(noRegion.Value);
+            }
```

Comment:
There can be projects with "Cannot determine" as their region so the assumption here is that we want to include those as well - in addition to the one matching the user region(s).

This is not specified in the AC of the original story so open to hear thoughts on whether to make this assumption or not

Reply:
On acq files the missing requirement was that cannot determine is not a valid option when selecting a region. May be worth asking if that is the case here as well.

Reply:
Well, ACQ and projects can be brought in via ETL and there's a valid case where region code would be 4 == Cannot determine.

I agree with your other PR that limits the regions upon creation so you cannot select region 4 but for searches I think we should still support region 4. Need a definitive answer from BAs